developer-extension/src/panel/components/tabs/eventsTab/eventRow.tsx

```diff
@@ -339,12 +339,16 @@ function LongTaskDescription({ event }: { event: RumLongTaskEvent }) {
 }
 
 function VitalDescription({ event }: { event: RumVitalEvent }) {
-  const vitalName = Object.keys(event.vital.custom!)[0]
-  const vitalValue = event.vital.custom![vitalName]
+  const vitalName = event.vital.name
+  const vitalValue = event.vital.duration
+  const vitalDetails = event.vital.details
   return (
     <>
-      Custom <Emphasis>{event.vital.type}</Emphasis> vital: <Emphasis>{vitalName}</Emphasis> of{' '}
-      <Emphasis>{vitalValue}</Emphasis>
+      Custom <Emphasis>{event.vital.type}</Emphasis> vital:{' '}
+      <Emphasis>
+        {vitalName} - {vitalDetails}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏If `vitalDetails` is empty do not add the separator ` - `